src/request_collector/cli.py

```diff
@@ -40,11 +40,28 @@
     multiple=True,
     help="Use this matrix server instead of the default ones. Include protocol in argument.",
 )
+@click.option(
```

Comment:
Can you move this into `common_options`?

Reply:
It looks like we are going to use 2 different disclaimers. Then this is not needed, right?